app/include/zmk/sensors.h

```diff
@@ -7,6 +7,6 @@
 #pragma once
 
 #define ZMK_KEYMAP_SENSORS_NODE DT_INST(0, zmk_keymap_sensors)
-#define ZMK_KEYMAP_HAS_SENSORS DT_NODE_EXISTS(ZMK_KEYMAP_SENSORS_NODE)
+#define ZMK_KEYMAP_HAS_SENSORS DT_NODE_HAS_STATUS(ZMK_KEYMAP_SENSORS_NODE, okay)
 #define ZMK_KEYMAP_SENSORS_LEN DT_PROP_LEN(ZMK_KEYMAP_SENSORS_NODE, sensors)
-#define ZMK_KEYMAP_SENSORS_BY_IDX(idx) DT_PHANDLE_BY_IDX(ZMK_KEYMAP_SENSORS_NODE, sensors, idx)
\ No newline at end of file
+#define ZMK_KEYMAP_SENSORS_BY_IDX(idx) DT_PHANDLE_BY_IDX(ZMK_KEYMAP_SENSORS_NODE, sensors, idx)
```

Comment:
Was this change meant to slip into this PR?

Reply:
Yeah, I did this as a small extra commit. It allows you to define the sensor node in the DT, *empty* so that user keymaps can just reference it, enable it, and set the list of sensors they want to use.

See the sample `bdn9_rev2.keymap` commented section near the top. Thoughts?

Reply:
I see. I think this makes sense. Might want a similar thing to be done on the tidbit.